extensions/target-specific/chromium/blink/blink-helpers/blink-helpers.js

```diff
@@ -345,20 +345,24 @@ Loader.OnLoad(function() {
         });
     }));
 
+    function formatLayoutUnit(layout_unit) {
+      return (layout_unit / 64) + "px";

```

Comment:
Ideally, we would have queried for "kFixedPointDenominator" or "kLayoutUnitFractionalBits" using DbgObject.constantValue so that we wouldn't have to hard code the number 64 here. However, those symbol names don't seem to be present in the pdbs, which likely means they have been optimized out during compilation. So this is fine, but could you add a comment clarifying that 64 signifies the fixed point denominator?

Reply:
Done